plugins/user/cronjobs

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor the number of cronjobs running per user
```

Comment:
Please use a perldoc header.

Reply:
Done. (I hope I'm using the correct `=head1` names?)

Reply:
Yes, they look good to me.